charts/service-deployment/values.yaml

```diff
@@ -74,6 +75,30 @@ hpa:
   # -- Average CPU utilization before auto-scaling starts
   averageCPUUtilization: 75
 
+keda:
```

Comment:
It feels like KEDA is ultimately a different way to trigger auto-scaling.. I am wondering if it makes more sense to nest `keda` inside of the `hpa` block and then make it configurable to either use `cpu` or `keda` within it?

This means we maintain the min/max replica commonality.  Wdyr?